notebooks/halos_rendering.ipynb

```diff
@@ -0,0 +1,457 @@
+{
```

Comment:
<p>A scatter plot showing with $z$ along the x-axis and $\log_{10}(\rm mass)$ along the y-axis will illustrate this table nicely.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sibirrer/sim-pipeline/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>